### ISSUE TITLE
fork to hass.io add-on?

### ISSUE THREAD

**Issue by jonand** [mylesgray/homebridge-blueair#23]
I'd like to control my new Blueair 480i via Home Assistant/hass.io. Unfortunately the hassio homebridge addon is [officialy dead](https://github.com/hassio-addons/addon-homebridge/issues/98). The newer HomeKit component seems to work quite well. Any chance homebridge-blueair could be forked to a new [hassio add-on](https://developers.home-assistant.io/docs/en/hassio_addon_index.html)?

**Comment by mylesagray** [comment]
Not sure what the ask is here - if there is a HomeKit component for hass.io - can it not interact with homebridge?

**Comment by jonand** [comment]
I don't believe hassio and homebridge can work together without the now dead hombridge hassio add-on.

What I'm asking is if your homebridge-blueair can be ported to an hassio add-on?

**Comment by mylesagray** [comment]
As far as I know, the dead plug-in was the opposite way round, it would export Hassio objects to HomeKit, where as the new HomeKit plugin for hassio allows HomeKit native (and I suppose homebridge too) to be imported into home assistant - so you could try that out?

https://www.home-assistant.io/components/homekit_controller/

I don’t plan on porting the plug-in any time soon as I don’t use home assistant for automations yet, only HomeKit.

**Comment by lensherm** [comment]
Hi guys. Any updates on integrating Blueair into Home Assistant?

**Comment by d3rw1n** [comment]
Would also love to see this! I no longer use homebridge so the integration of my Blueair 405 with Home Assistant can only be done through IFTTT webhooks

**Comment by lensherm** [comment]
I installed homebridge, add the blueair plugin and configured for it to see and succesfully control my Classic 480i. I proceeded to add the HomeKit Controller to Home Assistant and am able to see the air quality data and control the light. One thing I can't do is see the status or control the actual purifier fan from within Home Assistant, as there's no entity added for that. Any idea of how to proceed from here?

**Comment by zinogo** [comment]
> Would also love to see this! I no longer use homebridge so the integration of my Blueair 405 with Home Assistant can only be done through IFTTT webhooks

I used same method through IFTTT webhooks with hass.io currently. But I got the bad information, Blueair services are being discontinued on IFTTT on November 6, 2020. I really hope to have a hass.io addon for Blueair. Or some have workaround to add blueair homekit to hass.io. (Currently only can control light, hassio can't see air purifier.) Maybe have option change the type to fan for compatible to hassio as possible.

**Comment by d3rw1n** [comment]
> > Would also love to see this! I no longer use homebridge so the integration of my Blueair 405 with Home Assistant can only be done through IFTTT webhooks
> 
> I used same method through IFTTT webhooks with hass.io currently. But I got the bad information, Blueair services are being discontinued on IFTTT on November 6, 2020. I really hope to have a hass.io addon for Blueair. Or some have workaround to add blueair homekit to hass.io. (Currently only can control light, hassio can't see air purifier.) Maybe have option change the type to fan for compatible to hassio as possible.

I'm now stuck in the same situation as you. The IFTTT integration is officially dead. Was still working last night. Finally installed homebridge again after so long. Like you said unfortunately the HomeKit Controller component can't see an air purifier so no entity was created in Home Assistant.

**Comment by lensherm** [comment]
By the way, Node-Red does work with air purifiers on HomeBridge. A little kludgy, but it seems to be not too bad. Just need to find me a bit of time to finalize the flows in Node-Red to sync with corresponding entities in HA.
